src/main/java/com/nerdwin15/stash/webhook/PullRequestEventListener.java

```diff
@@ -62,16 +62,16 @@ protected void handleEvent(PullRequestEvent event) {
       return;
     }
 
-    String strRef = event.getPullRequest().getFromRef().toString()
-        .replaceFirst(".*refs/heads/", "");
+    String strRef  = "pr/" + Long.toString(event.getPullRequest().getId()) + "/from";
```

Comment:
What are the implications of this change? It looks we're changing the whole PR scheme here.